styles.css

```diff
@@ -62,9 +62,291 @@ body {
 .social-media-icons {
   height: 30px;
   display: flex;
+  margin: 8px;
 }
 
 .social-media-icons a {
   text-decoration: none;
-  margin: 8px;
+  align-items: center;
+  margin: 6px;
+}
+
+.social-media-icons img {
+  padding: 6px;
+}
+
+.works {
```

Comment:
Please, use the same background-color as in Figma:

Your design:
<img width="446" alt="image" src="https://user-images.githubusercontent.com/7243132/170822580-c53f3879-b869-4854-a345-0e2a322c6d16.png">

Figma:
<img width="446" alt="image" src="https://user-images.githubusercontent.com/7243132/170822607-62175401-a111-4f43-a25e-157a319d131c.png">